ddtrace/settings/_otel_remapper.py

```diff
@@ -0,0 +1,187 @@
+import os
+
+from ..constants import ENV_KEY
+from ..constants import VERSION_KEY
+from ..internal.logger import get_logger
+
+
+log = get_logger(__name__)
+
+
+OTEL_UNIFIED_TAG_MAPPINGS = {
+    "deployment.environment": ENV_KEY,
+    "service.name": "service",
+    "service.version": VERSION_KEY,
+}
+
+
+def _remap_otel_log_level(otel_value):
+    """Remaps the otel log level to ddtrace log level"""
+    if otel_value == "debug":
+        return "True"
+    else:
+        log.warning(
+            "ddtrace does not support otel log level '%s'. setting ddtrace to log level info.",
```

Comment:
should we state that we only support debug level?

Reply:
Done